scripts/zones/Selbina/npcs/Zaldon.lua

```diff
@@ -14,6 +14,49 @@ require("scripts/globals/titles")
 -----------------------------------
 local entity = {}
 
+entity.fishRewards =
```

Comment:
non-blocking:
I assume these rewards are varied based on era (COP, TOAU, etc)
Are all era based rewards gated by era fish obtain-ability?
If not, theres an opportunity to gate rewards based on expansion.

Reply:
Yes, they are based on era _but_ the fish to hand in won't be available if they are out of era. Content tagging this table did cross my mind also. The droprate on results doesn't seem to vary between eras. However, now the table is accessible, it would also be possible to change with era based module like in #2844